src/tools/melt/melt.cs

```diff
@@ -308,6 +308,30 @@ private static void MeltBinaryTable(Pdb inputPdb, InstallPackage package, string
             }
         }
 
+        private static bool ValidateMSIMatchesPdb(InstallPackage package, Pdb inputPdb)
+        {
+            const string KEY_PRODUCT_CODE = "ProductCode";
```

Comment:
Can't use static readonly at the class level and maintain the usage of the switch statement.  That said, it is a single check so I can easily switch that to an if statement.